docs/UserGuide.md

```diff
@@ -64,6 +64,8 @@ WedLog is a desktop app for wedding planning, optimized for use via a Command Li
 
 **Notes about the command format:** <br />
 
+* Words are defined to be any letters, numbers or special characters of length 1 or more
```

Comment:
Based on #214, I believe this still means that users who look the section on Address without seeing this small note will still perceive the single digit integer to be not very meaningful. Is it possible to just specify the acceptable values for Address directly as "any non blank values", or something along those lines? We can still keep this definition here for the purposes of the "alphanumeric words" of names, etc.